src/js/models/Col.js

```diff
@@ -0,0 +1,53 @@
+import { Record } from 'immutable'
+
+/**
+ * Default values for a new sign object
+ * 
+ * @static
+ * @constant
+ */
+const defaults = {
+  col_id: 0,
+  name: '',
+}
+
+/**
+ * Manage all the data related to a sign
+ * 
+ * Signs are immutable, and any mutations create new signs
+ * 
+ * @class
+ * @extends Record
+ */
+export default class Col extends Record(defaults) {
+
+  constructor(attrs) {
+    super(attrs)
+  }
+
+  /**
+   * @public
+   * @instance
+   * 
+   * @return {string} the sign Id
+   */
+  getID() {
```

Comment:
I'm chewing on how we can get some OO inheritance going on so we don't have to repeat this.